recipe/meta.yaml

```
@@ -13,7 +13,8 @@ source:
 
 build:
   number: {{ build }}
-  skip: true  # [((cuda_compiler_version == "None") and (linux64 or win)) or ((cuda_compiler_version != "10.2") and (ppc64le))]
+  string: "py{{ PY_VER }}h{{ PKG_HASH }}_{{ build }}_{{ opencl_impl }}"  # [osx]
```

To make khronos low priority
```suggestion
  string: "py{{ PY_VER }}h{{ PKG_HASH }}_{{ build }}_{{ opencl_impl }}"  # [osx]
  track_features:
    - openmm_opencl_khronos  # [opencl_impl == 'khronos']
```

---

Oh, nice! Thanks!